lib/spree_billing_sisow/engine.rb

```diff
@@ -22,9 +22,9 @@ def self.activate
 
     initializer "spree_billing_sisow.register.payment_methods", :after => 'spree.register.payment_methods' do |app|
       app.config.spree.payment_methods += [
-          Spree::BillingIntegration::SisowBilling::Ideal,
-          Spree::BillingIntegration::SisowBilling::Bancontact,
-          Spree::BillingIntegration::SisowBilling::Sofort
+          Spree::PaymentMethod::SisowBilling::Ideal,
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.